src/shared/components/simpleTable/SimpleTable.tsx

```diff
@@ -2,32 +2,34 @@ import * as React from 'react';
 import './styles.scss';
 import { If, Then,  Else } from 'react-if';
 
-type SimpleTableProps = {
+interface ISimpleTableProps {
     headers:JSX.Element[];
     rows:JSX.Element[];
     noRowsText?:string;
+    className?:string;
 };
 
-export default class SimpleTable extends React.Component<SimpleTableProps, {}>
+export default class SimpleTable extends React.Component<ISimpleTableProps, {}>
 {
 
 
     public render() {
+        const {headers, rows, className, noRowsText} = this.props;
 
-        const rows = this.props.rows.length > 0 ? this.props.rows :
+        const tableRows = rows.length > 0 ? rows :
             [<tr>
-                <td style={{textAlign:'center'}} colSpan={this.props.headers.length}>
-                    {this.props.noRowsText || "There are no results."}
+                <td style={{textAlign:'center'}} colSpan={headers.length}>
+                    {noRowsText || "There are no results."}
                 </td>
             </tr>];
 
         return (
-            <table className="table table-striped table-border-top">
+            <table className={`table table-striped table-border-top${className ? ' ' + className : ''}`}>
```

Comment:
https://www.npmjs.com/package/classnames  
this library makes it must easier to deal with conditional classes in jsx attribute